### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by ninecodes** [albingroen/prismabuilder.io#27]
Add missing accessibility features, when navigating with keyboard, reference for that would be [here](https://tailwindcss.com/docs/hover-focus-and-other-states#focus-visible)

Also resolve some lighthouse related topics to bring up the accessibility score:

![image](https://user-images.githubusercontent.com/31674307/145667845-bf443ea9-29c3-41be-982c-96c4fad86741.png)

**Comment by albingroen** [comment]
Now I believe I've done what I can do on my part to improve the a11y of this app. There are some things that aren't particularly accessible, like the dropdown components for example. These come from `@prisma/lens` though, so I can't really do too much about that, except create issues on their repo.
